client/openocd_rpc.py

```diff
@@ -28,7 +29,8 @@ def __init__(self,
                  tries=1,
                  verbose=False,
                  timeout=2,
-                 array_var_name='_rpc_array'):
+                 array_var_name='_rpc_array',
+                 ignore_regexps=[]):
```

Comment:
Nit: please change the default value to None and set the instance variable to [] if it is still None.  https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html